.github/dependabot.yml

```diff
@@ -1,16 +0,0 @@
-version: 2
-updates:
-  - package-ecosystem: "pip"
-    directory: "/"
-    schedule:
-      interval: "daily"
-
-  - package-ecosystem: "npm"
-    directory: "/"
-    schedule:
-      interval: "daily"
```

Comment:
Why not set it to `monthly` instead?

Reply:
Good point, `monthly` seems to strike a good balance. My major concern was that we have too many dependabot PRs that are just patch updates and don't add anything valuable. However, checking monthly will probably alleviate this problem.